chef/cookbooks/dhcp/recipes/default.rb

```diff
@@ -164,6 +164,6 @@
     end
   end
   supports :restart => true, :status => true, :reload => true
-  action :enable
+  action node[:provisioner][:enable_pxe] ? ["enable", "start"] : ["disable", "stop"]
```

Comment:
After a mumble chat with @rhafer , ```provisioner::dhcp_update.rb``` and ```provisioner::update_nodes.rb``` are enabled again. Should be changed in the next release.